modules/tinymce/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
+- Inline toolbars flicker when switching between editors #TINY-8594
```

Comment:
You also need a changelog entry for TINY-8503 if this is meant to fix that as well.

Reply:
@tiny-ben-tran the wrong thing was done here... TINY-8594 and TINY-8504 might be caused by the same underlying issue, but they are still different issues and need different changelog entries. TINY-8504 for example was about multiple toolbars rendering when using `fixed_toolbar_container` in inline mode.

Reply:
You mean TINY-8503 ticket? 
I think so too. I will ask Andy to verify `also fixed TINY-8503`.

Reply:
Yes sorry, hit the wrong key.